spec/dummy/app/assets/javascripts/application.js

```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
+// listed below.
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
+// or any plugin's vendor/assets/javascripts directory can be referenced here using a relative path.
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// compiled file. JavaScript code in this file should be added after the last require_* statement.
```

Comment:
Let's ignore the dummy app, as stated in https://github.com/ontohub/ontohub-models/blob/330c05fac30813a8c36987b3caddcd59e7447348/.rubocop.yml#L3